<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Authorization</class>
 <widget class="QDialog" name="Authorization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>208</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>539</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Вход в систему</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Загрузки/icon.png</normaloff>../../../Загрузки/icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="LoginAll_Layout">
     <item row="1" column="0">
      <layout class="QFormLayout" name="Login_Layout">
       <item row="3" column="0">
        <widget class="QLabel" name="Login_Label">
         <property name="text">
          <string>Имя пользователя:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="Login_Edit"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="Password_Edit">
         <property name="locale">
          <locale language="Russian" country="Russia"/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="TryLogin_Button">
         <property name="text">
          <string>Войти</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="Password_Label">
         <property name="text">
          <string>Пароль:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="Port_Edit">
         <property name="text">
          <string>5432</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="Port_Label">
         <property name="text">
          <string>Порт:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="Host_Edit"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="host_Label">
         <property name="text">
          <string>IP адрес:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>База данных:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="DataBase_Edit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
